scripts/init.d-xapissl

```diff
@@ -90,7 +90,7 @@ writeconffile () {
     echo "; Autogenerated by ${0}" > $SSLCONFFILE
     writec '; during xapi start-up.'
     writec '; '
-    if [ ${ANCIENT_STUNNEL} = 0 ]; then
+    if [ ${ANCIENT_STUNNEL} = 0 ] && [ x$CC_PREPARATIONS != 'xtrue' ]; then
```

Comment:
I think it would be better to use `x"$CC_PREPARATIONS" != x"true"` to highlight that the `x` is a dummy.

Reply:
👍   Will update, thanks!